src/docs/user_ai.md

```diff
@@ -289,10 +289,22 @@ To configure MCP servers, open the preferences and add entries to the `MCP Serve
   },
   "filesystem": {
     "command": "npx",
-    "args": ["-y", "@modelcontextprotocol/server-filesystem", "/Users/YOUR_USERNAME/Desktop"],
+    "args": [
+      "-y",
+      "@modelcontextprotocol/server-filesystem",
+      "/Users/YOUR_USERNAME/Desktop"
+    ],
     "env": {
       "CUSTOM_ENV_VAR": "custom-value"
     }
+  },
+  "git": {
+    "command": "uvx",
+    "args": [
+      "mcp-server-git",
```

Comment:
```suggestion
    "args": [
      "run",
      "mcp-server-git",
```

Reply:
I believe this is required at least from my local testing.

Reply:
So we your updated config it work for you?

Reply:
So my config looks like this:

```
"git": {
      "command": "uvx",
      "args": [
          "mcp-server-git",
          "--repository",
          "/home/eugen/Git/theia"
      ]
  }
```

I installed uv and uvx is in the path:
```bash
> uvx --version
> uv-tool-uvx 0.6.0
```

Reply:
I just added both configs now

Reply:
```
> uv --version
uv 0.5.28
```
```
> uvx --version
uvx 2.5.1
```
Looks different on my side